neurokit2/data/database.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import zipfile
+from pathlib import Path
```

Comment:
Same here can we do `import pathlib` and then `pathlib.Path` (I like to see where functions come from ^^)

Reply:
I am the same way and did this in my previous code but then I saw pretty much everyone else using `from pathlib import Path` and so switched for the sake of convention, up to you though

Reply:
yeah let's be iconoclast and internally consistent ^^